src/ompl/base/src/OptimizationObjective.cpp

```diff
@@ -98,6 +103,11 @@ ompl::base::Cost ompl::base::OptimizationObjective::combineCosts(Cost c1, Cost c
     return Cost(c1.value() + c2.value());
 }
 
+ompl::base::Cost ompl::base::OptimizationObjective::subtractCosts(Cost c1, Cost c2) const
+{
+    return Cost(c2.value() - c1.value());
+}
+
```

Comment:
I dislike how the operators are in reverse order for this call - intuition would say that it would return `c1 - c2`.